File: Folder Java/Kode/GorOlahraga.java
// GorOlahraga.java
class GorOlahraga extends TempatUsaha {
    private String jenisOlahraga;
    private String fasilitas;
    private String skala;
    private double hargaSewa;

    public GorOlahraga() {}
    
    public GorOlahraga(String namaTempat, String noIzin, String kategoriUsaha, String lokasi, String deskripsi,
                       String jenisOlahraga, String fasilitas, String skala, double hargaSewa) {
        super(namaTempat, noIzin, kategoriUsaha, lokasi, deskripsi);
        this.jenisOlahraga = jenisOlahraga;
        this.fasilitas = fasilitas;
        this.skala = skala;
        this.hargaSewa = hargaSewa;
    }
}

File: Folder Python/Kode/GorOlahraga.py
# GorOlahraga.py
from TempatUsaha import TempatUsaha

class GorOlahraga(TempatUsaha):
    def __init__(self, nama_tempat="", no_izin="", kategori_usaha="", lokasi="", deskripsi="",
                 jenis_olahraga="", fasilitas="", skala="", harga_sewa=0.0):
        super().__init__(nama_tempat, no_izin, kategori_usaha, lokasi, deskripsi)
        self.__jenis_olahraga = jenis_olahraga
        self.__fasilitas = fasilitas
        self.__skala = skala
        self.__harga_sewa = harga_sewa